server.js

```diff
@@ -28,6 +28,7 @@ app.disable('x-powered-by');
 app.listen(port);
 
 console.log(`Server is running at http://127.0.0.1:${port}`);
+console.log(`Server is running`,host_dir);

```

Comment:
The meaning of this logging message seems confusing.

Reply:
This line was deleted